activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Don't call commit/rollback callbacks despite a record isn't saved.
```

Comment:
If I understand the PR right I think we can simplify the changelog entry.

```diff
-*   Don't call commit/rollback callbacks despite a record isn't saved.
+*  Don't call commit/rollback callbacks when a record isn't saved.
```

Reply:
Applied at 20b94af9eb9305d19a343f72f0afb18eb49e2de7.